tests/xharness/Logging/Log.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace xharness.Logging {
+
+	public abstract class Log : TextWriter, ILog {
+		public ILogs Logs { get; private set; }
+		public string Description { get; set; }
+		public bool Timestamp { get; set; } = true;
+
+		public static string XML_LOG = "XML log";
+		public static string NUNIT_RESULT = "NUnit results";
```

Comment:
Do you need these fields when you have also the enum in `ILog` and the corresponding `ToString` extension?

Reply:
I added the enums, forgot to change it, will update!